server/midddleware/middleware.go

```diff
@@ -71,9 +70,10 @@ func Get(config *config.Config, tenantMgr *metadata.TenantManager, txMgr *transa
 		grpc_ratelimit.StreamServerInterceptor(&RateLimiter{}),
 		grpc_auth.StreamServerInterceptor(authFunction),
 		namespaceInitializer.NamespaceSetterStreamServerInterceptor(),
-		grpctrace.StreamServerInterceptor(grpctrace.WithServiceName(util.Service)),
-		grpc_logging.StreamServerInterceptor(grpc_zerolog.InterceptorLogger(sampledTaggedLogger), []grpc_logging.Option{}...),
-		validatorStreamServerInterceptor(),
+	}
+
+	if config.DatadogTrace.Enabled {
+		streamInterceptors = append(streamInterceptors, ddTraceStream())
 	}
 
 	if config.Metrics.Grpc.Enabled && config.Metrics.Grpc.ResponseTime {
```

Comment:
What's our goal with the "basic" GRPC interceptor? Do we still need it? Or is this in place to make the transition smoother?